lib/library.rb

```diff
@@ -0,0 +1,35 @@
+#require 'Book'
+require 'Date'
+
+class Library
+ attr_accessor :books  # :duedate  #TODO:status
+
+  def initialize(library)
+    @books = []
+  #  @duedate = set_due_date
+  end
+
+ def check_in(book) #TODO library if its needed or not !!!
+    add = {item: book, status: :okay}
+    @books.push add
+  end
+
+  def lend(title,person)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `person`. If it's necessary, use `_` or `_person` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)